portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java

```diff
@@ -17,6 +17,11 @@
 import com.google.sps.data.Comment;
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
```

Comment:
Let's keep these alphabetized https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing.